src/Kodify/BlogBundle/Resources/views/Post/view.html.twig

```diff
@@ -9,5 +9,19 @@
         <div class="panel-body">
             {{ post.content }}
         </div>
+        <div>
+            <div class="panel-heading"><h4>Comments</h4></div>
+            {% if comments is empty %}
+                <div class="panel-body">There are no comments on this post, let's create some!!</div>
+            {% else %}
+                {% for comment in comments %}
+                    <div class="panel-body">
+                        <div class="pull-right">by: {{ comment.author.name }} on: {{ comment.createdAt|date("m/d/Y") }}</div>
+                        <div>{{ comment.content }}</div>
+                    </div>
+                {% endfor %}
```

Comment:
We could certainly use pagination here. Maybe maintain the post always visible at the top of the page along with the "create comment" button, and create a different section below for comments including pagination and organizing them from most recent at the top of the section to oldest ones at the bottom.